### ISSUE TITLE
I keep crashing

### ISSUE THREAD

**Issue by scigeotech** [UpcraftLP/PeachDudes-1.10.2#1]
Whenever I start up my stuff, it crashes after loading

> ---- Minecraft Crash Report ----
> // Oh - I know what I did wrong!

Time: 9/22/16 3:56 PM
Description: Rendering screen

java.lang.NullPointerException: Rendering screen
    at net.minecraftforge.fml.common.versioning.DefaultArtifactVersion.containsVersion(DefaultArtifactVersion.java:74)
    at net.minecraftforge.fml.common.versioning.DefaultArtifactVersion.equals(DefaultArtifactVersion.java:56)
    at net.minecraftforge.fml.common.versioning.Restriction.toStringFriendly(Restriction.java:211)
    at net.minecraftforge.fml.common.versioning.VersionRange.toStringFriendly(VersionRange.java:498)
    at net.minecraftforge.fml.client.GuiModsMissing.func_73863_a(GuiModsMissing.java:106)
    at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:382)
    at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1094)
    at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1076)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:371)
    at net.minecraft.client.main.Main.main(SourceFile:124)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
## A detailed walkthrough of the error, its code path and all known details is as follows:

-- Head --
Thread: Client thread
Stacktrace:
    at net.minecraftforge.fml.common.versioning.DefaultArtifactVersion.containsVersion(DefaultArtifactVersion.java:74)
    at net.minecraftforge.fml.common.versioning.DefaultArtifactVersion.equals(DefaultArtifactVersion.java:56)
    at net.minecraftforge.fml.common.versioning.Restriction.toStringFriendly(Restriction.java:211)
    at net.minecraftforge.fml.common.versioning.VersionRange.toStringFriendly(VersionRange.java:498)
    at net.minecraftforge.fml.client.GuiModsMissing.func_73863_a(GuiModsMissing.java:106)
    at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:382)

-- Screen render details --
Details:
    Screen name: net.minecraftforge.fml.client.GuiModsMissing
    Mouse location: Scaled: (0, 239). Absolute: (0, 0)
    Screen size: Scaled: (427, 240). Absolute: (854, 480). Scale factor of 2
Stacktrace:
    at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1094)
    at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1076)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:371)
    at net.minecraft.client.main.Main.main(SourceFile:124)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
    Minecraft Version: 1.10.2
    Operating System: Windows 10 (x86) version 10.0
    Java Version: 1.8.0_101, Oracle Corporation
    Java VM Version: Java HotSpot(TM) Client VM (mixed mode), Oracle Corporation
    Memory: 141633040 bytes (135 MB) / 402157568 bytes (383 MB) up to 523501568 bytes (499 MB)
    JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx512M -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
    IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
    FML: MCP 9.32 Powered by Forge 12.18.1.2092 20 mods loaded, 20 mods active
    States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
    U    mcp{9.19} [Minecraft Coder Pack](minecraft.jar)
    U    FML{8.0.99.99} [Forge Mod Loader](forge-1.10.2-12.18.1.2092.jar)
    U    Forge{12.18.1.2092} [Minecraft Forge](forge-1.10.2-12.18.1.2092.jar)
    U    tntmod{1.7.10} [TNTmod]([1.10]TNTMod.jar)
    U    appleskin{1.0.4} [AppleSkin](AppleSkin-mc1.10.2-1.0.4.jar)
    U    cannibalism{3.0.0} [Cannibalism](Cannibalism-3.0.0-1.10.2.jar)
    U    claycraft{3.0} [Claycraft](Claycraftmod v3.0 - MC1.10.2 Forge.jar)
    U    cordage{1.0.1} [Cordage](cordage-1.0.1.jar)
    U    csaddle{1.2} [Craftable Saddles](Craftable Saddles 1.2.jar)
    U    DragonMounts{r46-1.10.2} [Dragon Mounts](DragonMounts-r46-1.10.2.jar)
    U    lootablebodies{2.2.0} [DrCyano's Lootable Bodies](DrCyanosLootableBodies_1.10.2-2.2.0.jar)
    U    F2L{1.10.2} [Flesh2Leather](Flesh2Leather [1.10.2].jar)
    U    samsfooddetails{1.0.1} [Food Details](FoodDetails-1.10.2-1.0.1.jar)
    U    foragecraft{1.7} [ForageCraft](ForageCraft-1.7.jar)
    U    fishing{1.4.4} [Fishing Net Mod](GiacomosFishingNet1.10.2-1.4.4.jar)
    U    JEI{3.10.0.261} [Just Enough Items](jei_1.10.2-3.10.0.261.jar)
    U    Neat{1.3-12} [Neat](Neat 1.3-12.jar)
    U    pplus{1.10.2} [Peaceful Plus](Peaceful Plus [1.10.2].jar)
    U    peachdudes{0.1.0-b2} [Peach Dudes Mod](peachdudes-1.10.2-0.1.0-b2.jar)
    U    xlfoodmod{1.10.2-0.3.0} [XL Food Mod](XL Food Mod-1.10.2-0.3.0.jar)
    Loaded coremods (and transformers):
    GL info: ' Vendor: 'Intel' Version: '4.3.0 - Build 20.19.15.4331' Renderer: 'Intel(R) HD Graphics 4400'
    Launched Version: 1.10.2-forge1.10.2-12.18.1.2092
    LWJGL: 2.9.4
    OpenGL: Intel(R) HD Graphics 4400 GL version 4.3.0 - Build 20.19.15.4331, Intel
    GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

```
Using VBOs: Yes
Is Modded: Definitely; Client brand changed to 'fml,forge'
Type: Client (map_client.txt)
Resource Packs: Occupied.zip
Current Language: English (US)
Profiler Position: N/A (disabled)
CPU: 4x Intel(R) Core(TM) i3-4030U CPU @ 1.90GHz
```

**Comment by UpcraftLP** [comment]
Could you add the mod files or provide download links? (All links you added are 404...)

**Comment by scigeotech** [comment]
i didnt add the links on purpose. i dont know why they are identified as links

**Comment by UpcraftLP** [comment]
Oh okay then, could you still add your mods to the crash report so I can try and reproduce the error?
Btw it seems you're missing [tca-core](https://minecraft.curseforge.com/projects/tca-core) (the coremod for this mod).

**Comment by UpcraftLP** [comment]
Closing this, SHOULD be fixed in v0.2.0-b3.
